src/test/resources/features/list.feature

```diff
@@ -1,8 +1,8 @@
 Feature: List actions
 
-  @ArchiveList @LogOut
-  Scenario: New list creation
-    Given I log in as admin user
-    When I select TestBoard board
-    And I create a new list with "TestList" as title
-    Then I should see the new created list with the given title
\ No newline at end of file
+#  @ArchiveList @LogOut
+#  Scenario: New list creation
+#    Given I log in as admin user
+#    When I select TestBoard board
+#    And I create a new list with "TestList" as title
+#    Then I should see the new created list with the given title
```

Comment:
Please add an empty line at the end of the class.

Reply:
Solved